<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paciente</class>
 <widget class="QMainWindow" name="Paciente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paciente</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Señor paciente, por favor responda de manera honesta a todas las preguntas realizadas a continuación.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Nota: El resultado dependera de la veracidad de estas.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Marque las opciones que sean verdaderas, deje en blanco las que no.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Preguntas</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;¿Algun familiar que padesca cancer?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;¿Padece algun tipo de cancer?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Sexo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="masculino">
         <property name="text">
          <string>Masculino</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QSpinBox" name="edad"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;¿Presenta alguno de estos sintomas?:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="femenino">
         <property name="text">
          <string>Femenino</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Edad&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Registro familiar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Su dia a dia incluye&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="cancermama">
         <property name="text">
          <string>Cancer de mama</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="cancerrinon">
         <property name="text">
          <string>Cancer de riñon</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="cancercolon">
         <property name="text">
          <string>Cancer de colon</string>
         </property>
        </widget>
       </item>
       <item row="23" column="0">
        <widget class="QCheckBox" name="cancerpulmon">
         <property name="text">
          <string>Cancer de pulmon</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="memoria">
         <property name="text">
          <string>Perdida de la memoria</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="nauseas">
         <property name="text">
          <string>Nauseas</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="vista">
         <property name="text">
          <string>Problemas devisión y/o audición</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="vomito">
         <property name="text">
          <string>Vómitos</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="comportamiento">
         <property name="text">
          <string>Cambios en el comportamiento</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="convulsiones">
         <property name="text">
          <string>Convulsiones</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="debilidad">
         <property name="text">
          <string>Debilidad en un lado del cuerpo</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="dolor_cabeza">
         <property name="text">
          <string>Dolor de cabeza</string>
         </property>
        </widget>
       </item>
       <item row="23" column="1">
        <widget class="QCheckBox" name="movimiento">
         <property name="text">
          <string>Dificultad para controlar el movimiento</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="abuelos">
         <property name="text">
          <string>Abuelos, Abuelas</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="tios">
         <property name="text">
          <string>Tios, Tias, Primos</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="padres">
         <property name="text">
          <string>Madre, Padre, Hermanos/as</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QCheckBox" name="checkBox_12">
         <property name="text">
          <string>Exposicion a radiacion</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QCheckBox" name="checkBox_15">
         <property name="text">
          <string>Estres constante</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QCheckBox" name="checkBox_21">
         <property name="text">
          <string>Golpes severos/constantes en la cabeza</string>
         </property>
        </widget>
       </item>
       <item row="23" column="2">
        <widget class="QCheckBox" name="checkBox_22">
         <property name="text">
          <string>Medicacion constante</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="enviarButton">
      <property name="text">
       <string>Enviar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
